applications/dashboard/scss/editor/embeds/_embedGlobal.scss

```diff
@@ -99,14 +101,18 @@
 // Edit Mode changes
 .richEditor { // Only in edit mode
```

Comment:
Avoid nesting as much as possible. These are brand new classes, so you shouldn't need to. 

It's unclear to me why there are 3 sections in `_embedText.scss`. They should either have unique classes or split into 3 partials.

Reply:
Done